manim/utils/rate_functions.py

```diff
@@ -1,4 +1,8 @@
-"""A selection of rate functions, i.e., *speed curves* for animations."""
+from math import sqrt
+
+
+"""A selection of rate functions, i.e., *speed curves* for animations.
+Please find a standard list at https://easings.net/"""
```

Comment:
Nitpick: you usually want the module docstring to be the first thing in the file.